webapp/junit.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuites name="jest tests" tests="2" failures="0" time="3.353">
+  <testsuite name="undefined" errors="0" failures="0" skipped="0" timestamp="2020-08-10T13:35:00" time="2.462" tests="2">
+    <testcase classname=" Plugin manifest, id and version are defined" name=" Plugin manifest, id and version are defined" time="0.002">
+    </testcase>
+    <testcase classname=" Plugin id and version are defined" name=" Plugin id and version are defined" time="0.001">
+    </testcase>
+  </testsuite>
+</testsuites>
```

Comment:
Was this file added intentionally?

Reply:
No, there were missing entries in `webapp/.gitignore` that help defend against committing these files. I added the `.gitignore` to fix it, but I had not deleted the existing committed files.